### PR TITLE
add leaflet-realtime v1.3.0 with npm auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #6746 
@x09326  please help me review this PR, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/perliedman/leaflet-realtime
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/leaflet-realtime
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 181
   - Count of watchers: 21
   - Count of forks: 36
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 27
   - Downloads in the last month: 196

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc @perliedman

### CODE REVIEW

File: ajax/libs/leaflet-realtime/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "realtime"
+  ],
+  "author": {
+    "name": "Per Liedman",
+    "email": "per@liedman.net"
+  },
+  "license": "ISC",
+  "browserify-shim": {
+    "leaflet": "global:L"
+  },
```
Comment (x09326) [inline ajax/libs/leaflet-realtime/package.json]: @PeterDaveHello 
I'm not sure whether ```"browserify-shim"``` is needed.

Comment (perliedman) [inline ajax/libs/leaflet-realtime/package.json]: Without it, you will bundle your own instance of Leaflet when building the plugin, which is rarely what you want.

Comment (PeterDaveHello) [inline ajax/libs/leaflet-realtime/package.json]: hmmmm ... actually we don't use this info on cdnjs, @perliedman so isn't the file `leaflet-realtime.js` under `dist` folder standalone?

Comment (perliedman) [inline ajax/libs/leaflet-realtime/package.json]: @PeterDaveHello the `dist` is standalone, but `browserify-shim` is needed to build it properly.

I mean, `package.json` isn't required at all if you're just using the `dist` file?

Comment (PeterDaveHello) [inline ajax/libs/leaflet-realtime/package.json]: Thanks @perliedman, in fact we don't manually build libraries, we only host pre-built files from upstream, thanks!

**Comment by PeterDaveHello** [comment]
Once this PR pass the peer review, it'll need a rebase on the latest master branch and review again, since it's 4 days ago.

**Comment by x09326** [comment]
@kennynaoh 
Please rebase the branch with the latest master branch.

**Comment by kennynaoh** [comment]
@x09326 
I have rebased to latest master branch.
Please help me review it again.
Thanks.

**Pull request merged**